builtin/logical/pki/path_revoke.go

```diff
@@ -22,8 +22,11 @@ hyphen-separated octal`,
 			},
 		},
 
-		Callbacks: map[logical.Operation]framework.OperationFunc{
-			logical.UpdateOperation: b.metricsWrap("revoke", noRole, b.pathRevokeWrite),
+		Operations: map[logical.Operation]framework.OperationHandler{
+			logical.UpdateOperation: &framework.PathOperation{
+				Callback: b.metricsWrap("revoke", noRole, b.pathRevokeWrite),
+				// This should never be forwarded. See backend.go for more information.
```

Comment:
@stevendpclark Should this be updated to mirror the Tidy operation endpoint, and forward this to the primary node of the cluster since it may write in the case that the cert wasn't set to no-store?

Reply:
I think this is better, just leave the lower layers within Vault do their thing. We might as well get rid of the `ForwardPerformanceStandby: true` on the tidy operations while we are at it, it's not incorrect but again let the lower levels of Vault do their thing.

Reply:
From our conversation, I've left this one and the Tidy one alone, but updated the `crl/rotate` operation to forward automatically.